### PR TITLE
Upload 18 BAT tests

### PR DESCRIPTION
**Pull request by wangji1**
Upload first version of stable bat tests, there are 2 common tests
to check librealsense and realsense_camera installed, and there are
4 specific tests of each camera (R200/F200/SR300/ZR300) to check
camera info matrix, enabled tf, set power off/on and force power
off/on, totally 18 tests.

All the tests are based on rospy, using python script as test type.

**Please ensure the above *guidelines for contributing* are met.**

Fixes Issue: #

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: realsense_camera/CMakeLists.txt [hunk 1]
```diff
@@ -110,6 +110,17 @@ if (CATKIN_ENABLE_TESTING)
...
 endif()
 
+set (SCRIPT_DIR ${CATKIN_DEVEL_PREFIX}/lib/${PROJECT_NAME})
+if (CATKIN_ENABLE_TESTING)
+  file(COPY test/files/scripts DESTINATION ${SCRIPT_DIR})
+endif()
+
+if (CATKIN_ENABLE_TESTING)
+  add_custom_target(clean-script
+    rm -rf ${SCRIPT_DIR}/scripts
```
Comment (mdhorn) [inline realsense_camera/CMakeLists.txt]: Why is this needed?
I'm not exciting about having an `rm -rf` in the build flow.

**Review by mdhorn — CHANGES_REQUESTED** [review]

**Comment by mdhorn** [comment]
@wangji1 
As we are adding new Python scripts to the repository, please enable roslint for Python and ensure it is passing.

```
diff --git a/realsense_camera/CMakeLists.txt b/realsense_camera/CMakeLists.txt
index 77d5e909b801..1d3fb979ada4 100644
--- a/realsense_camera/CMakeLists.txt
+++ b/realsense_camera/CMakeLists.txt
@@ -59,6 +59,7 @@ generate_dynamic_reconfigure_options(
 
 # ROS Lint the code
 roslint_cpp()
+roslint_python()
 roslint_add_test()
 
 #################################

```

And use this command to run roslint:
`catkin_make roslint`

@nwang2 Possibly you can help Jinliang with resolving the roslint failures.

File: realsense_camera/CMakeLists.txt (hunk 1 above)
Comment (wangji1) [reply realsense_camera/CMakeLists.txt]: I want to remove the copied scripts when user clean the building environment, but not find how to remove it when run "catkin_make clean". so I add a customer target to remove them.
What it the correct method?

**Comment by wangji1** [comment]
I have updated python scripts, and they are passed catkin_make roslint.
@mdhorn  I'm not sure will you still want to merge this PR to RS camera project?
If yes, please help to review again.
If not, please close it directly.

Thanks!

**Review by mdhorn — APPROVED** [review]

**Pull request merged**